src/modules/Auth/containers/signup/EmailForm/useEmailForm.tsx

```diff
@@ -38,8 +43,11 @@ export const useEmailForm = () => {
       });
       return;
     } else {
-      registerReq({ email }, referrerUser?.id)
-        .then(() => localStorage.setItem('email', email))
+      registerReq({ email }, referrerUser?.id, event_id)
+        .then(() => {
+          localStorage.setItem('email', email);
+          event_id && setEventsFilter();
+        })
```

Comment:
For my information: Why are you saving filters here? Are you going to redirect user to he search page after sign-up?